inifile.cc

```diff
@@ -91,11 +91,12 @@ open_gpsbabel_inifile()
   char* envstr;
   gbfile* res = NULL;
 
+  // TODO(viettaml): use _wgetenv() on Windows, convert to QString.
```

Comment:
you would hope qgetenv would handle this for us, but the doc explictly warns that it won't work.

Reply:
Yeah, I looked into this as well and it looks like it might be fixed in the very latest Qt 5.10, but that doesn't help us (or me, as Earth uses Qt 5.4): https://bugreports.qt.io/browse/QTBUG-41006

The best thing to do would be to implement a wrapper, say xgetenv() in defs.h/utils.cc, doing the right conversion for each OS and returning a QString.

Your thoughts on whether or not that should go in a separate follow-up PR?

Reply:
If you will please do that in a separate PR.